test/plugins/metrics.test.js

```diff
@@ -139,6 +143,7 @@ describe('request metrics plugin', function() {
     it('should return metrics with use error', function(done) {
         SERVER.on('uncaughtException', function(req, res, route, err) {
             assert.ok(err);
+            res.send(err);
```

Comment:
Same question regarding `'uncaughtException'` and the need for `res.send()` than above.

Reply:
see comment above